docs/api/settings.md

```diff
@@ -0,0 +1,283 @@
+REST API: Settings
+==================
+
+The settings REST API allows you to view and update WooCommerce Stripe settinngs.
+The endpoints, requests, and responses have similar structure with [WooCommerce core setting options REST API](https://woocommerce.github.io/woocommerce-rest-api-docs/?shell#setting-options).
```

Comment:
I would say "share a similar structure with" or "have a similar structure to" but I am not English native - it just sounds better. Not very important suggestion.